auth-and-authz/README.md

```diff
@@ -64,63 +62,69 @@ For instance, if we want to let user `app101` bind to the exchange `app100-x-eve
            dc=example,                                       dc=example,            dc=example,      dc=example,
            dc=com                                            dc=com                 dc=com           dc=com
             |
-  +---------+----------------------+   
+  +---------+----------------------+
   |         |                      |
 cn=users,   cn=administrator       cn=app100-x-events-read
  ou=dev,     ou=dev,                ou=dev,
  ou=env,     ou=env,                ou=env,
  dc=example, dc=example,            dc=example,
  dc=com      dc=com                 dc=com
    ||           ||                     ||
----------     --------              --------        
-cn=app100,..   cn=admin-dev,...      cn=app101,....  
+---------     --------              --------
+cn=app100,..   cn=admin-dev,...      cn=app101,....
 cn=app101,..
 cn=app102,..
 cn=admin-dev,...
 ```
 
 From top to bottom:
   1. At the Root/top is our organization.
-  2. From it hangs:  
-    - the environments/vhosts under `ou=env, ...`  
-    - all the users/apps under `ou=People,...`  
+  2. From it hangs:
```

Comment:
I cannot remove those two 2 trailing spaces because otherwise Github will not render them in separate lines.

Reply:
You can push changes to my branch if you'd like. GitHub allows it for pull requests.